app/controllers/katello/api/v2/activation_keys_controller.rb

```diff
@@ -243,12 +243,6 @@ def index_relation
       activation_keys
     end
 
-    def find_activation_key
```

Comment:
When using find_resource we should not have to use our own find_<resourcename> methods.